FlightAxis/flightaxis.py

```diff
@@ -37,11 +43,20 @@ class FlightAxis:
     PX4_DEVICE = 'tcpin:localhost:4560'
     TIMEOUT_S = 0.05
 
-
     LAT_0 = 45.25608036722125 # [deg]
     LON_0 = -122.84116451943059 # [deg]
     ALT_0 = 36 # [m]
     NANOTESLA_TO_GAUSS = 0.01
+    MAG_RECALC_THRESHOLD_DEG = 0.1 # Just a guestimate now
+
+    def getHeader(self) -> list:
+        header = []
+        for key in self.getKeytable().keys():
+            header.append(key)
+        for idx in range(FlightAxis.RC_CHANNLES):
+            s = f"rc_channel_{idx}"
+            header.append(s)
```

Comment:
```suggestion
header = [key for key in self.getKeytable()] + [f"rc_channel_{idx}" for idx in range(FlightAxis.RC_CHANNLES)]
```
Depending on the type of getKeytable it would be even easier to `list(self.getKeytable())`. Not all iterators may allow this.